### PR TITLE
Use 'Inclusive Organization' action to invite contributors to organization

### PR DESCRIPTION
**Pull request by lekterable**
this need a `ACCESS_TOKEN` secret created to work

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+jobs:
+  invite:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Invite contributor to the organization
+        uses: lekterable/inclusive-organization-action@v1.0.0
+        with:
+          organization: styled-components
+        env:
+          ACCESS_TOKEN: ${{ secrets.ACCESS_TOKEN }}
```
Comment (mxstbr) [inline .github/workflows/main.yml]:
```suggestion
          ACCESS_TOKEN: ${{ secrets.ORG_ADMIN_ACCESS_TOKEN }}
```

**Comment by lekterable** [comment]
ready to merge 🤞 🤞

**Comment by mxstbr** [comment]
This is just missing one tiny thing I just noticed: when we auto-invited a person previously, Aeryn posted a nice comment explaining that they were invited and the reasoning behind it.

Can we add that to the action too? This is what we used to post: https://github.com/styled-components/styled-components/pull/1652#issuecomment-380669357

```
Thank you so much for helping us improve styled-components! Based on our Community Guidelines every person that has a PR of any kind merged is offered an invitation to the styled-components organization—that is you right now!

Should you accept, you'll get write access to the main repository. (and a fancy styled-components logo on your GitHub profile!) You'll be able to label and close issues, merge pull requests, create new branches, etc. We want you to help us out with those things, so don't be scared to do them! Make sure to read our contribution and community guidelines, which explains all of this in more detail. Of course, if you have any questions just let me know!
```

**Comment by lekterable** [comment]
I like the idea! I should be able to work on it a bit later today, do you want to wait for that or should we merge this and we can update the version add a comment in another PR?

**Comment by mxstbr** [comment]
Let's merge this first and then you can submit another PR here once we have comment support! That sound good?

**Comment by mxstbr** [comment]
We can also immediately test if it works: when we merge this, you should get an invite 😜

**Comment by mxstbr** [comment]
Let's see if it works 🤞 https://github.com/styled-components/styled-components/runs/294203874

**Comment by mxstbr** [comment]
Seems to not have errored?! Did you get an invite???

**Comment by mxstbr** [comment]
It worked!! :tada:

Screenshot 2019-11-08 at 12 01 12

I just now realised that we need to do another thing: newly invited folks need to be added to the [Collaborators team](https://github.com/orgs/styled-components/teams/collaborators) otherwise they do not have write access to any repo.

Another option to add to the action? :angel:

**Comment by lekterable** [comment]
of course I did, I'm a pro 😎

**Comment by lekterable** [comment]
I'll work on it today!

**Pull request merged**